content/concept-amendments.md

```diff
@@ -179,33 +182,54 @@ An address with a SignerList can disable the master key even if a regular key is
 * `tefNOT_MULTI_SIGNING`
 * `tefBAD_AUTH_MASTER`
 
-<!--{# to be added in a later version:
+
 ## OwnerPaysFee
 
-| Amendment ID | Status |
-|--------------|--------|
-| TBD | Coming soon |
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 9178256A980A86CF3D70D0260A7DA6402AAFE43632FDBCB88037978404188871 | In development. |
+
+Fixes an inconsistency in the way [transfer fees](concept-transfer-fees.html) are calculated between [OfferCreate](reference-transaction-format.html#offercreate) and [Payment](reference-transaction-format.html#payment) transaction types. Without this amendment, the holder of the issuances pays the transfer fee if an offer is executed in offer placement, but the initial sender of a transaction pays the transfer fees for offers that are executed as part of payment processing. With OwnerPaysFee, the holder of the issuances always pays the transfer fee, regardless of whether the offer is executed as part of a Payment or an OfferCreate transaction. Offer processing outside of payments is unaffected.
```

Comment:
Nitpick: change "Without this amendment" in the second sentence to "Without OwnerPaysFee". Or "With OwnerPaysFee" to "With this amendment" to make them parallel. Unless there's a good reason not to that I'm not thinking of.